Foundation.hs

```diff
@@ -134,6 +134,9 @@ embedLayout widget = do
     pc <- widgetToPageContent widget
     withUrlRenderer $(hamletFile "templates/embed-layout-wrapper.hamlet")
 
+analytics :: App -> Maybe (Entity User) -> url -> Html
```

Comment:
Actually, this is the render-url function. So the name is bad anyway. Maybe just `a` would be better.